testsuite/features/reposync/allcli_update_activationkeys.feature

```diff
@@ -158,7 +158,6 @@ Feature: Update activation keys
     And I wait for child channels to appear
     And I select the parent channel for the "proxy_container" from "selectedBaseChannel"
     And I wait for child channels to appear
-    And I check "openSUSE Leap Micro 5.5 (x86_64)"
```

Comment:
We might have a similar issue of ambiguity with:
```
And I check "Uyuni Client Tools for openSUSE Leap Micro 5.5 (x86_64)"
And I check "Uyuni Client Tools for openSUSE Leap Micro 5.5 (x86_64) (Development)"
```

Reply:
I ran the test (see above) and the test suite did not have any issues with those 2.